test/LMO_test.jl

```diff
@@ -217,3 +218,113 @@ diffi = x_sol + 0.3 * rand([-1, 1], n)
     @test sum(isapprox.(x, x_sol, atol=1e-6, rtol=1e-2)) == n
     @test isapprox(f(x), f(result[:raw_solution]), atol=1e-6, rtol=1e-3)
 end
+
+# Test for L2normBallBLMO
+@testset "L2normBall BLMO continuous" begin
+    n = 20
+
+    # Generate a solution inside the L2 ball (||x|| <= 1)
+    x_sol = randn(rng, n)
+    x_sol = x_sol ./ (norm(x_sol) * 1.5)
+
+    # Round some coordinates to integers for testing
+    num_int = 5
+    int_indices = sort(rand(rng, 1:n, num_int))
+    for idx in int_indices
+        x_sol[idx] = 0
+    end
```

Comment:
How can you guarantee that `x_sol` will stay inside the ball? If you cannot, the tests later will fail even if we find the correct solution.